src/components/banners/Banner.tsx

```diff
@@ -15,6 +15,17 @@ export type BannerProps = {
   message: ReactNode;
   pinned?: boolean;
   intense?: boolean;
+  // text for showMore link
+  showMoreLinkText?: ReactNode;
+  // text for showless link
+  showLessLinkText?: ReactNode;
+  // color for show more links
+  showMoreLinkColor?: string;
+  // additionalMessage is shown next to message when clicking showMoreLinkText.
+  // disappears when clicking showLess link
+  // note that this additionalMessage prop is used to determine show more/less behavior or not
+  // if undefined, then just show normal banner with message
+  additionalMessage?: ReactNode;
```

Comment:
Can you put `additionalMessage` above the other showMore props, because it is what activates the feature? Thanks!

Reply:
@chowington Yep, I just did it

Reply:
@chowington adding hover effect required a bit more works than I initially thought, but anyway it is added now.

Reply:
Yeah, you sometimes have to add state if you want dynamic CSS...thanks DK!